react-native/react/actions/tracker.js

```diff
@@ -84,10 +86,40 @@ export function onFollowHelp (username: string): Action {
   }
 }
 
-export function onCloseFromActionBar (username: string): Action {
-  return {
-    type: Constants.onCloseFromActionBar,
-    payload: {username}
+function trackUser (username: string, state: {tracker: RootTrackerState}): Promise<boolean> {
```

Comment:
yeah in this case we do, the promise was in case you cared if it was successful or not